packages/hardhat-chai-matchers/src/changeEtherBalance.ts

```diff
@@ -0,0 +1,93 @@
+import { BigNumber, BigNumberish, providers } from "ethers";
+import { ensure } from "./calledOnContract/utils";
+import { Account, getAddressOf } from "./misc/account";
+import { BalanceChangeOptions } from "./misc/balance";
+
+export function supportChangeEtherBalance(Assertion: Chai.AssertionStatic) {
+  Assertion.addMethod(
+    "changeEtherBalance",
+    function (
+      this: any,
+      account: Account | string,
+      balanceChange: BigNumberish,
+      options: BalanceChangeOptions
```

Comment:
Shouldn't this be

```suggestion
      options?: BalanceChangeOptions
```

?